Autocomplete/Sources/Controllers/ZPLSuggestionWindowController.m

```diff
@@ -90,6 +91,7 @@ - (instancetype)init {
     _tableView.delegate = self;
     _tableView.dataSource = self;
     _tableView.allowsEmptySelection = NO;
+    _tableView.backgroundColor = [NSColor zpl_backgroundColor];
```

Comment:
I'm not sure if we should set it to `nil` here. We should only update the color if `zpl_backgroundColor` is not `nil`. Same for other uses.